OpenTrons/opentrons-api
# fix(labware-library): make labware creator accessible via external links

# Overview
This PR adds an HTML resource for labware creator that will get served when hitting labware.opentrons.com/create. all this HTML file does is route users to the correct hash route that gets picked up by react router.

This is necessary because we have external links that users use to access labware creator directly without going through the top level of labware library. 

It also replaces labware library's browser router with a hash router so that requests to `/create` do not get sent to the server, since labware creator is served client side via react router. 

Fixes https://opentrons.atlassian.net/browse/RESC-339

## Test Plan and Hands on Testing

- Go to http://sandbox.labware.opentrons.com/ll_update-build-base and make sure the page loads correctly (CSS will be messed up, this will be fixed by https://github.com/Opentrons/opentrons/pull/16502
- Navigate to LC (if you scroll to the bottom of the page theres a button that should be clickable), make sure the page loads
- In a new browser window, hit http://sandbox.labware.opentrons.com/ll_update-build-base/create directly (without going through the root of LL). Make sure the page loads correctly

## Changelog

- Add top level resource for Labware Creator so it can be accessed via external links

## Review requests

See test plan above

## Risk assessment

Medium, but also fixes a bug
